### ISSUE TITLE
Sortable issues

### ISSUE THREAD

**Issue by woutor** [darklow/django-suit#12]
Hi darklow,

Thanks for django-suit! It's a nice admin theme and I appreciate your approach using Bootstrap. I am glad you're working on the Sortable feature, but unfortunately found two issues:
- Using SortableTabularInline for inlines, if the main object does not extend SortableModelAdmin, the following exception occurs while editing an object:

AttributeError:
type object 'ModelForm' has no attribute 'Meta'

in suit/admin.pu r.54:
    self.form.Meta.widgets[self.sortable] = SortableListForm.Meta.widgets[
- Using SortableTabularInline with extra inlines (extra > 0), and submitting an object with empty 'extra' inlines:

Form validation error on empty inline objects.

.. probably because the SortableTabularInline sets the sortable field?

Hope you can take a look at this!

**Comment by realsby** [comment]
I am getting the same error.. 

- Django (1.10.5)
- django-suit (0.2.23)
- Python 2.7.6

Admin.py
```
class OverrideQuestionInline(SortableStackedInline):
    model = Question
    extra = 1
    sortable = 'order'
    suit_classes = 'suit-tab suit-tab-question'


class QuestionGroupAdmin(SortableModelAdmin):
    inlines = [OverrideQuestionInline, ]
    list_display = ('title', 'order')
    list_editable = ('order',)
    search_fields = ['question__question', 'question__answer', 'title']
    fieldsets = [
        (None, {
            'classes': ('suit-tab', 'suit-tab-general',),
            'fields': ['title', ]
        }),
    ]

    suit_form_tabs = (('general', 'General'), ('question', 'FAQS'),)
```

Model.py

```
class QuestionGroup(models.Model):
    title = models.CharField(max_length=50, blank=False, null=False)
    order = models.PositiveIntegerField(null=True, blank=True)
    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True)

    def __unicode__(self):
        return '%s' % (self.title,)

    class Meta:
        ordering = ['order']


class Question(models.Model):
    question_group = models.ForeignKey(QuestionGroup, blank=False, null=False)
    question = models.TextField(blank=False, null=False)
    answer = models.TextField(blank=False, null=False)
    order = models.PositiveIntegerField(null=True, blank=True)
    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True)

    def __unicode__(self):
        return '%s' % (self.id,)

    class Meta:
        ordering = ['order']
```

**Comment by darklow** [comment]
@realsby what error? Please post full stack trace.

**Comment by realsby** [comment]
Sorry for my english..

Using SortableTabularInline with extra inlines (extra > 0), and submitting an object with empty raise empty validation errors for fields which have blank=False, null=False settings..

**Comment by darklow** [comment]
@realsby You are not on latest version. Please update to Django Suit `0.2.24` where the bug was fixed #566:
https://github.com/darklow/django-suit/blob/develop/CHANGELOG.rst#v0224-2017-01-27
